lib/dereference.js

```diff
@@ -96,6 +96,9 @@ function crawl(obj, path, pathFromRoot, parents, $refs, options) {
 function dereference$Ref($ref, path, pathFromRoot, parents, $refs, options) {
   debug('Dereferencing $ref pointer "%s" at %s', $ref.$ref, path);
 
+  if (process.env.SCHEMAS && !/^https?:\/\//i.test($ref.$ref) && !/^\#/.test($ref.$ref))
+    path = process.env.SCHEMAS;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'path'.](https://www.codacy.com/app/jamesmessinger/json-schema-ref-parser/file/2557103956/issues/source?bid=2491852&fileBranchId=3292219#l100)